examples/tutorials/analysis-3d/event_sampling.py

```diff
@@ -117,10 +115,9 @@
 
 check_tutorials_setup()
 
-
 ######################################################################
 # Define an Observation
-# ~~~~~~~~~~~~~~~~~~~~~
+# ---------------------
```

Comment:
This was needed to make the notebook render correctly in jupyter, it should also work in sphinx...